src/cljs/main/broadfcui/common/table/table.cljs

```diff
@@ -0,0 +1,141 @@
+(ns broadfcui.common.table.table
```

Comment:
We're not going to convert every table as part of this PR, but it will move over once we finish migrating all of the tables.